static/locales/zh-TW.json

```
@@ -443,6 +503,48 @@
     "screenshotTitle": "點擊查看截圖 {{index}}",
     "removeScreenshot": "移除此截圖"
   },
+  "emotion": {
+    "manager": "Live2D 情感映射管理器",
+    "description": "管理和設定Live2D角色的動作與表情映射",
+    "selectModel": "選擇Live2D模型：",
+    "model": "模型: {{model}}",
```

Biome (2.3.14)

[error] The key uploadExecutionError was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)